ui/src/shared/components/cells/Cell.scss

```diff
@@ -247,6 +247,7 @@ $cell--header-button-active-color: $c-pool;
 
   &.time-series-container--alert-check {
     padding-right: $ix-marg-b + 30px;
+    padding-bottom: $ix-marg-b + 17px;
```

Comment:
If this `padding-bottom` is intended to create space for the visibility toggles I'd suggest creating a variable in `_variables.scss` and using it here as well as the filter controls to ensure the 2 share a height value